cd/Official/logic1.ocd

```diff
@@ -149,6 +149,37 @@ for all x | x and not(x) = false
 </OMOBJ></FMP>
 </CDDefinition>
 
+<CDDefinition>
+<Name>nand</Name>
+<Role>application</Role>
+<Description> 
+This symbol represents the logical nand function which is an n-ary
+function taking boolean arguments and returning a boolean value. It
+is false if all arguments are true or true otherwise.
+</Description>
+<CMP>
+for all x | x nand not(x) = true
+</CMP>
+<FMP><OMOBJ xmlns="http://www.openmath.org/OpenMath" version="2.0" cdbase="http://www.openmath.org/cd">
```

Comment:
The example is good, but maybe we can also have a more "definitional one"
\forall x,y,z nand(x,y,z) = not (and(x,y,z)). 
This would also be good for the other ones.

Reply:
Yes, particularly for xnor.  I also think xnor needs an addition to the CMP, since there seem to be alternative definitions on the lines of (x1/\x2/\.../\xn)\/(!x1/\!x2/\.../\!xn), i.e. "all the same".